src/towncrier/test/test_packaging.py

```diff
@@ -1,22 +1,16 @@
 # Copyright (c) Amber Brown, 2015
 # See LICENSE for details.
 
-from incremental import Version
 from twisted.trial.unittest import TestCase
 
-from towncrier._version import _hatchling_version
+import towncrier
 
 
 class TestPackaging(TestCase):
-    def test_version_warning(self):
+    def no_version_attr(self):
         """
-        Import __version__ from towncrier returns an Incremental version object
-        and raises a warning.
+        towncrier.__version__ was deprecated, now no longer exists.
```

Comment:
not sure if `towncrier.__version__` was considered public or private API.

For private API, raising AttributeError is ok.
If it's public API, we need to raise a separate error with an info message.